pkg/storage/mvcc_history_test.go

```diff
@@ -954,6 +956,36 @@ func cmdInitPut(e *evalCtx) error {
 	})
 }
 
```

Comment:
Given that the client currently cannot pass the ImportEpoch to MVCCPut(), should we worry about stats updates?

Reply:
That's fine, we can drop stats assertions here. Let's generalize it as `put_raw`.

It would be better to pass this through `MVCCPut`, either via a separate parameter or a new struct, but we can add a `TODO` for it for now.